compiler-rt/test/memprof/Unit/lit.site.cfg.py.in

```diff
@@ -0,0 +1,22 @@
+@LIT_SITE_CFG_IN_HEADER@
+
+import os
+import platform
+import re
+import shlex
+
+# Load common config for all compiler-rt unit tests.
+lit_config.load_config(config, "@COMPILER_RT_BINARY_DIR@/unittests/lit.common.unit.configured")
+
+# Setup config name.
+config.name = 'MemProfiler-Unit'
+config.target_arch = "@arch@"
+assert config.target_arch == 'x86_64'
+
+config.test_exec_root = os.path.join("@COMPILER_RT_BINARY_DIR@",
+                                     "lib", "memprof", "tests")
+
+config.test_source_root = config.test_exec_root
+
```

Comment:
The asan version of this file has the following, which looks like it controls the amount of parallelism in some cases. Since memprof uses shadow memory too (not sure about ctx_profile), we should probably include this here?

```
if not config.parallelism_group:
  config.parallelism_group = 'shadow-memory'
```

Reply:
done for memprof, ctx_profile doesn't use shadow memory.